docs/commands.md

```diff
@@ -677,7 +677,15 @@ _Arguments_
 
 _Return_
 
-An array with alternating entries that represent the following key-value pairs:
+For a module genernal information: An array with alternating entries that represent the following key-value pairs:
+
+* **Version**: a string showing the current module version.
+* **Low level API Version**:  a string showing the current module's low level api version.
+* **RDB Encoding version**: a string showing the current module's RDB encoding version.
+* **TensorFlow version**: a string showing the current loaded TesnorFlow backend version.
+* **ONNX version**: a string showing the current loaded ONNX Runtime backend version.
```

Comment:
What about pytorch?

Reply:
see in the description, we are waiting for a PR to expose this on TORCH sie